src/sentry/sentry_metrics/visibility/metrics_blocking.py

```diff
@@ -165,32 +173,40 @@ def unblock_tags_of_metric(metric_mri: str, tags: Set[str], projects: Sequence[P
     )
 
 
-def get_blocked_metrics(projects: Sequence[Project]) -> Mapping[int, BlockedMetrics]:
-    blocked_metrics_by_project = {}
+def get_metrics_blocking_state(projects: Sequence[Project]) -> Mapping[int, MetricsBlockingState]:
+    metrics_blocking_state_by_project = {}
 
     for project in projects:
-        blocked_metrics_by_project[project.id] = BlockedMetrics.load_from_project(
+        metrics_blocking_state_by_project[project.id] = MetricsBlockingState.load_from_project(
             project=project, repair=False
         )
 
-    return blocked_metrics_by_project
+    return metrics_blocking_state_by_project
 
 
-def get_blocked_metrics_for_relay_config(project: Project) -> BlockedMetricsRelayConfig:
+def get_metrics_blocking_state_for_relay_config(
+    project: Project,
+) -> Optional[MetricsBlockingStateRelayConfig]:
     try:
-        blocked_metrics = get_blocked_metrics([project])[project.id]
+        metrics_blocking_state = get_metrics_blocking_state([project])[project.id]
     except MalformedBlockedMetricsPayloadError as e:
         sentry_sdk.capture_exception(e)
         # In case of a malformed configuration, we will notify Sentry and return no metrics, since it's an unrecoverable
         # situation, unless we want to force overwrite the config.
-        return BlockedMetricsRelayConfig(deniedNames=[])
-
-    # For now, we just return the metric mris of the blocked metrics. Once tags are supported in Relay, we will return
-    # also the blocked tags for the metric.
-    return BlockedMetricsRelayConfig(
-        deniedNames=[
-            blocked_metric.metric_mri
-            for blocked_metric in blocked_metrics.metrics.values()
-            if blocked_metric.is_blocked
-        ]
-    )
+        return None
```

Comment:
Decided to return `None` so that on the other side, the config will not be appended to the project configuration, making it slimmer.